persistence-api/src/main/java/org/apache/cassandra/stargate/db/MutationEvent.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.cassandra.stargate.db;
+
+import org.apache.cassandra.stargate.schema.TableMetadata;
+
+/** Represents a change made in the database. */
+public interface MutationEvent {
```

Comment:
can we introduce an intermidiete interface between  `MutationEvent` -> `DeleteEvent` AND `RowUpdateEvent` smth like:
```
class PartitionedMutationEvent{
   List<CellValue> getPartitionKeys();
    List<CellValue> getClusteringKeys();
}
```
?
Those methods are in both implementations, and having a common interface will make the CDC producer logic easier